src/llamafactory/extras/constants.py

```diff
@@ -596,6 +596,33 @@ def register_model_group(
 )
 
 
+register_model_group(
+    models={
+        "Index-1.9B-Chat": {
+            DownloadSource.DEFAULT: "IndexTeam/Index-1.9B-Chat",
+            DownloadSource.MODELSCOPE: "IndexTeam/Index-1.9B-Chat",
+        },
+        "Index-1.9B-Character": {
```

Comment:
rename to `Index-1.9B-Charater-Chat` for the auto chat template https://github.com/hiyouga/LLaMA-Factory/blob/bd08b8c441c47076faa03cc1efde21b22f14f058/src/llamafactory/extras/constants.py#L121